sys/kern/vfs_mount.c

```diff
@@ -420,7 +420,7 @@ kern_nmount(struct thread *td, struct iovec * __capability iovp, u_int iovcnt,
 
 	AUDIT_ARG_FFLAGS(flags);
 	CTR4(KTR_VFS, "%s: iovp %p with iovcnt %d and flags %d", __func__,
-	    (void *)(__cheri_addr intptr_t)iovp, iovcnt, flags);
+	    (void *)(uintptr_t)(uintcap_t)iovp, iovcnt, flags);
```

Comment:
Dropping the cast and using `%lp` might be better.

Reply:
indeed